modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+Storage class defaults in {VirtProductName} are configured to provide ReadWriteOnce (RWO) access mode
+by default. Therefore, LiveMigration is not enabled by default unless you
+alter the access mode of your virtual machine.
+
+.Prerequisites
+* In order to use LiveMigration, virtual machines must have a persistent volume claim (PVC) with a shared
+ReadWriteMany (RWX) access mode to be live migrated.
+
+.Procedure
+To override the default RWO access mode, update the storage class configuration
+by doing the following:
+
+. Export the storage class object.
++
+----
+$ export STORAGE_CLASS="nfs"
+----
++
+. Update the storage class `kubevirt-storage-class-defaults` from RWO to RWX access mode.
```

Comment:
`kubevirt-storage-class-defaults` is the name of the ConfigMap

Reply:
Changed to:

To override the default RWO access mode, update the *ConfigMap* 
`kubevirt-storage-class-defaults` storage class to use RWX 
access mode.